Configuration/TCA/tx_academicpersons_domain_model_organisational_unit.php

```diff
@@ -75,11 +75,6 @@
                 'type' => 'passthrough',
             ],
         ],
-        'his_id' => [
```

Comment:
Property, getter and setter is not removed from corresponding `Fgtclb\AcademicPersons\Domain\Model\OrganisationalUnit` Domain
Model.

Which could be breaking. I would emphasize this using

```
[!!!][TASK]
```

prefix for commit and pull-request.